pex/pex.py

```diff
@@ -407,17 +407,21 @@ def _execute(self):
       return self.execute_interpreter()
 
   def execute_interpreter(self):
-    if sys.argv[1:]:
+    if len(sys.argv) < 2:
+      import code
+      code.interact()
+    elif len(sys.argv) > 2 and sys.argv[1] == '-m':
+      mod = sys.argv[2]
+      sys.argv = sys.argv[2:]
```

Comment:
`sys.argv` should not be modified here.

Reply:
noting that `sys.argv` is modified below in the existing code too.. but I don't think that's necessary either. mind yanking it too while you're in here?

Reply:
actually.. scratch that.. I see why the `sys.argv` modification is needed now (sorry about the thrash).

Reply:
I'm actually not sure what the reason for `sys.argv` modification was, so I was just trying to match the behavior below. If a different modification is needed here, let me know and I can make that change.